### ISSUE TITLE
Traditional Chinese (Taiwan) Translation

### ISSUE THREAD

**Issue by asoolin** [lenmus/lenmus#109]
Hello, I came from to help translate Lenmus into traditional chinese. This is probably the first time of me ever using Github so I'm not sure of what to post. I guess we follow up on this issue for problems?

**Comment by cecilios** [comment]
> I guess we follow up on this issue for problems

Yes, that's the idea. But not only for problems, also for questions, help, etc. Instead of using emails everything is in here and you can later find if required. Also, anyone involved in the project is aware of what is going on and can participate.

**Comment by cecilios** [comment]
At http://lenmus.org/en/phonascus/translation_instructions you can find translation instructios, but do not download the zip package from SourceForge, as it is not updated. Instead,  use the latest PO files in the lenmus repo. Current PO files for eBooks form are at https://github.com/lenmus/langtool/tree/master/locale/zh_CN

To download any of them click on it. The file content will be displayed in a new window. Click on the [Raw] button just over the first line of the file, at right. Now the file content is displayed in your browser. Save that page Menu > File > Save Page As  and give it the correct name (e.g. `L2_MusicReading_zh_CN.po`).

As to the PO file for the GUI it is at https://github.com/lenmus/lenmus/tree/master/locale/zh_CN . File lenmus_zh_CN.po

You should also review the `study-guide.htm` file.

**Comment by asoolin** [comment]
Do all my new translations need to be marked as "proofread"? Also in what context does one use "Change color property?"

**Comment by cecilios** [comment]
> sorry about this amount of questions

No problem. you are welcome

> Do all my new translations need to be marked as "proofread"?

I don't know if this is possible with PO files. In any case, it will be just a mark for you (if you decide to continue updating the translation when a new release), or for other translator who, probably, would like to review it even if marked as "proofread". Short answer: is up to you.

> Also in what context does one use "Change color property?

The user is editing a document or score, and would like to change the color of an object (e.g. a paragraph or a group of music notes in the score).

> TOC stands for table of content?

Yes

> Virtual keyboard as in keyboard for piano?

No, it is a computer keyboard. Will be used for tablets.

> does %d have any meaning in translations?

`%d, %s, <a>, </a>` and other marks are very important and have to be maintained in the translation. Marks starting with % are place holders for a value that will be later introduced by the program. %d refers to a number and %s refers to a string. For instance: `"There are %d apples"` could result in `"There are 5 apples"`, and sentence `"There are %s apples"` might produce `"There are five apples"`.

Marks such as `<a>` and `</a>` go always in pairs enclosing some words. In the translation, they must enclose the same translated words. For instance, sentence `"This text is in <b>bold</b> face"` could be translated into `"Deze tekst is <b>vetgedrukt</b>"`

> some of the translations are wrong, and it shows that the original English text has been altered, whats the problem with that?

poEdit and other translation tools sometimes try to help the translator by selection an existing translation for a "similar" English sentence, so that the work of typing is smaller. But many many times the suggested translation is garbage.

**Comment by asoolin** [comment]
85% complete/confirmed
one question today, for period do i use 。 or .
5/21
